### DISCUSSION TITLE
github pages new page downloads something issue

### DISCUSSION THREAD

**Discussion by yabberish** [community/community#21665] [Other Feature Feedback, Questions, & Ideas]
<p>This has been happening for me for a while now, everytime I create a new page, and I try to enter it, it keeps downloading something, and I can’t enter the page, please help, thanks!</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/elflanded">@elflanded</a>.</p>
<p>Thanks for being here! Would you mind sharing your repo? That’ll be helpful in trying to figure out what’s going on.</p>

**Comment by yabberish** [comment]
<p>Hey there!</p>
<p>My file is here: <a href="https://github.com/Elflanded/blogs/blob/master/news/tech/altertrail-hiring.md" rel="nofollow noopener">https://github.com/altertrail/altertrail.github.io/blob/master/job-info.md</a></p>
<p>whenever I try to access the page, it always downloads the text that I put.</p>
<p>the link is, <a href="https://altertrail.github.io/job-info" rel="nofollow noopener">https://altertrail.github.io/job-info</a></p>
<p>thanks!</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Loads OK for me although the web page doesn’t have the formatting of the .md file.</p>
<p>[Some of your tags are messed up, and “view source” looks like:</p>
<pre><code>&lt;p1&gt; Currently, that is all we are hiring. &amp;lt;/p&amp;gt;
</code></pre>
<p>According to the Chrome tools, the only downloads I see are:</p>
<ol>
<li>
<p>the CSS file</p>
</li>
<li>
<p>three fonts</p>
</li>
</ol>
<p>Perhaps try a built-in font rather than downloading?</p>
